docs/src/utils/track.ts

```diff
@@ -1,5 +1,9 @@
 // https://github.com/aws-amplify/docs/blob/main/src/utils/track.ts
 let configured = false;
+let firstPageOfVisit = true;
+let AWSCShortbread;
+let s;
+let AWSMA;
```

Comment:
Is this being used?

Reply:
Nope, removed. I was before I changed it to `globalThis`